strings.go

```diff
@@ -129,6 +132,7 @@ const PLAYLIST_SKIPPED_HTML = `
 const HELP_HTML = `<br/>
 	<b>User Commands:</b>
 	<p><b>!help</b> - Displays this help.</p>
+	<p><b>!search (yt|sc) query</b> - Search on Youtube or Soundcloud for a query and add first hit.</p>
```

Comment:
As stated above, it may be better to change "yt" and "sc" to "youtube" and "soundcloud".